public/core.d.ts

```diff
@@ -11086,6 +11086,23 @@ declare namespace JMap {
      * ```
      */
     function getJmapIdAsIntegerIfPossible(id: any): JId
+
+    /**
+     * **JMap.Util.asyncProcess**
+     *
+     * Run the parameter function after one or multiple delays.
+     *
+     * @param fn The function to run
+     * @param timeoutsInMilliseconds An array of delay in milliseconds
+     * @throws if fn is not a function, if timeoutsInMilliseconds is not a non-empty array of number
```

Comment:
le param s'appelle "callback"

@param **_callback_** The function to run
@throws if **_callback_** is not a function ......